.gitignore

```diff
@@ -9,4 +9,7 @@ coverage
 # Macos
 .DS_Store
 
-lerna-debug.log
\ No newline at end of file
+lerna-debug.log
```

Comment:
Please revert those changes, those files should be committed. Since you are not changing any dependency your PR should only contains SCSS and JS changes. If the other changes on lock files comes from installing / building the app locally please still ignore them as dependencies updates are typically done on maintenance releases

Reply:
Oops. I added them to .gitignore, but they still weren't ignored. Will fix.

Reply:
Oops. Right, thanks.